source/core/GenericFunctions.cpp

```diff
@@ -304,15 +333,32 @@ InstructionCore* EmitGenericBinOpInstruction(ClumpParseState* pInstructionBuilde
                         void *savedArg = pInstructionBuilder->_argPointers[0];
                         if (pInstructionBuilder->_argumentState == ClumpParseState::kArgumentNotResolved) {
                             // invoked via recursion from Generic array/cluster binop
-                            pInstruction = pInstructionBuilder->EmitInstruction(&convertOpToken, 2,
-                              sourceYType, pInstructionBuilder->_argPointers[1], destType, (void*)-2);
+                            pInstruction = pInstructionBuilder->EmitInstruction(
+                                &convertOpToken,
+                                2,
+                                sourceYType,
+                                pInstructionBuilder->_argPointers[1],
+                                destType,
+                                (void*) - 2);
                             // tell vector/cluster op invoking us to convert the 2nd arg
                         } else {
-                            pInstruction = pInstructionBuilder->EmitInstruction(&convertOpToken, 2,
-                              sourceYType, pInstructionBuilder->_argPointers[1], destType, destArg);
+                            pInstruction = pInstructionBuilder->EmitInstruction(
+                                &convertOpToken,
+                                2,
+                                sourceYType,
+                                pInstructionBuilder->_argPointers[1],
+                                destType,
+                                destArg);
                         }
-                        pInstructionBuilder->EmitInstruction(&savedOperation, 3, sourceXType, savedArg,
-                                                            destType, destArg, destType, destArg);
+                        pInstructionBuilder->EmitInstruction(
+                            &operationName,
+                            3,
+                            sourceXType,
+                            savedArg,
+                            destType,
+                            destArg,
+                            destType,
+                            destArg);
```

Comment:
How about:

`pInstructionBuilder->EmitInstruction(&savedOperation, 3, `
`        sourceXType, savedArg,  `
`        destType, destArg,  `
`        destType, destArg);  `
Keeping types and args they go with together, at least.

Reply:
I see what you mean. If this function signature ever changed we would have to go back and updated all of the callers just for spacing, so I'm just undoing this for now.